### PR TITLE
NAS-118499 / 22.12 / Extracting some of VM dialogs into separate components

### PR DESCRIPTION
**Pull request by undsoft**
Test the following actions on the VMs page:
- Stopping
- Cloning
- Deleting
- Opening a display device (VM must be on).
- Opening a display device when there are multiple display devices in a VM
- Opening a display device when it is password protected.

### CODE REVIEW

**Review by denysbutenko — CHANGES_REQUESTED** [review]
I was redirected `/ui/vm` after opening a display device when VM has a single or multiple display devices. Tested on m40 instance

File: src/app/pages/vm/vm-list/delete-vm-dialog/delete-vm-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      [tooltip]="helptext.delete_dialog.zvolsTooltip | translate"
+    ></ix-checkbox>
+    <ix-checkbox
+      formControlName="force"
+      [label]="'Force Delete?' | translate"
+      [tooltip]="helptext.delete_dialog.forceTooltip | translate"
+    ></ix-checkbox>
+    <ix-input
+      formControlName="confirmName"
+      [label]="'Enter {name} below to confirm.' | translate: { name: vm.name }"
```
Comment (denysbutenko) [inline src/app/pages/vm/vm-list/delete-vm-dialog/delete-vm-dialog.component.html:16]: There is a similar string that may have translations:
`"Enter {value} below to confirm."`

**Comment by undsoft** [comment]
@denysbutenko I think url redirection is an artefact of localhost testing. If you enable 'Preserve Log' on Network tab in Chrome inspector and [enable Auto-open DevTools for Popups](https://stackoverflow.com/questions/12212504/automatically-open-chrome-developer-tools-when-new-tab-new-window-is-opened), you can catch it:
Снимок экрана 2022-10-10 в 11 36 02

**Review by denysbutenko — APPROVED** [review]
Looks good 👍🏻

**Pull request merged**